pkg/cri/resource-manager/policy/builtin/static-plus/static-plus-policy.go

```diff
@@ -19,7 +19,7 @@ import (
 	"fmt"
 	"strconv"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nitpick: I'd understand the explicit naming if you called it corev1 to be in line with metav1, but I fail to see any benefit here if it's explicitly named the same as the default.

Reply:
They seem to come automagically from golint... fixed them to `corev1` to keep the tooling happy.